### PR TITLE
Financial circuit readme upgrades

### PR DESCRIPTION
**Pull request by Ybittan**
This PR makes updates to the readme of the financial circuit service to give context to the purpose of the repo, as well as what it does, how it can be used, and where it is headed in the future. Also various spelling fixes throughout the repo

### CODE REVIEW

File: financial-circuit-service/README.md [hunk 1]
```diff
@@ -22,6 +27,10 @@ make run
 
 3. **Run Postman Collection**
 
+```
 Import futurevalue.postman_collection.json into postman and send requests
 
 Can be found under internal/api/collection
+```
+
+**Note**: Currenty the circuit does not support number of years input greater than 18. There is potential for future upgrades if the need is found.
```
Comment (Kasshern) [inline financial-circuit-service/README.md:36]: Currently

File: financial-circuit-service/README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Future Value Circuit Service
-Go (Golang) API REST Service with Gin Framework and Gnark Zero-Knowledge Fututure Value Circuit
+Go (Golang) API REST Service with Gin Framework and Gnark Zero-Knowledge Future Value Circuit
+
+This folder serves as a fully functioning service that allows the user to generate and verify zero-knowledge proofs through the API.
+The ZK circuit takes in the 4 necessary values for calculating the future value (Future Value, Present Value, Interest Rate, and Number of years). The circuit will then prove that the user correctly calculated the future value, given the other inputs, and generate a proof and proving key. Anyone can then take that proof and proving key, and provide it to the prover function along with the other public values (Interest Rate and Number of Years) and the proving key, and cryptographically prove the correctness of the end value, without needing the private future and present values.
```
Comment (Kasshern) [inline financial-circuit-service/README.md:5]: "Anyone can then take that proof and proving key, and provide it to the prover function along with the other public values (Interest Rate and Number of Years) and the proving key, and cryptographically prove the correctness of the end value, without needing the private future and present values. "   This sentence seems to mention proving key twice. Also I think at this point in the flow they are providing this information to the verifying function?

**Review by Kasshern — CHANGES_REQUESTED** [review]
One spelling mistake and one sentence that seems kinda confusing that I commented on.

**Review by markymark-26 — APPROVED** [review]
lgtm

**Comment by markymark-26** [comment]
@GoldenBit0 pls review

**Review by Kasshern — APPROVED** [review]

**Pull request merged**